### DISCUSSION TITLE
Structure/Process for Working Groups

### DISCUSSION THREAD

**Discussion by jqnatividad** [ckan/ckan#8064] [Ideas]
As part of Phase II of the [NSF POSE](https://new.nsf.gov/funding/opportunities/pathways-enable-open-source-ecosystems-pose) project (https://civicdataecosystem.org) for CKAN, we proposed to convene several "communities of practice" to work on key initiatives (e.g. DCAT 3, Water Data standards, etc.), with POSE providing the necessary "scaffolding" to support these communities/working groups. 

One idea is to follow the W3C protocol as described in detail here:

https://www.w3.org/2023/Process-20231103/

One concern with the W3C protocol is that it may be too "heavy-weight".

Another is to use a process similar to Rust's Request for Comments process:

https://rust-lang.github.io/rfcs/0002-rfc-process.html

A concern with the Rust process is that its targeted to "technical" issues.

Are there any other processes/protocols that can be considered to allow both technical and non-technical Subject Matter Experts to participate?

**Comment by thegostev** [comment]
Hi @jqnatividad thanks for sharing. We need these working groups so that we might implement result of these activities at the send part of 2024.

I like W3C process document for it comprehensiveness. It's not exactly what you asked for, rather a thinking pieces around CoP: https://hbr.org/2000/01/communities-of-practice-the-organizational-frontier and https://www.pmi.org/learning/library/communities-practice-advancing-ways-people-together-7840 .
